src/main/archetype/all/pom.xml

```diff
@@ -66,8 +66,8 @@
 #if ( $isSpaProject )
                         <embedded>
                             <groupId>com.adobe.aem</groupId>
-                            <artifactId>spa.project.core.core</artifactId>
-                            <target>/apps/${appId}-vendor-packages/install</target>
```

Comment:
I think you actually need both `spa.project.core.core` and `spa.project.core.ui.apps` otherwise you won't get the HierarchyPageImpl from the core

Reply:
I think this was working because the spa ui.apps includes the spa core bundle.

Reply:
Yeah the project's ui.apps used to embed it. Now this looks good to me..